virtcontainers/clh.go

```diff
@@ -877,45 +856,61 @@ func (clh *cloudHypervisor) getAvailableVersion() error {
 
 func (clh *cloudHypervisor) LaunchClh() (string, int, error) {
 
-	errStr := ""
-
 	clhPath, err := clh.clhPath()
 	if err != nil {
 		return "", -1, err
 	}
 
 	args := []string{cscAPIsocket, clh.state.apiSocket}
 	if clh.config.Debug {
-
-		logfile, err := clh.logFilePath(clh.id)
-		if err != nil {
-			return "", -1, err
-		}
-		args = append(args, cscLogFile)
-		args = append(args, logfile)
+		args = append(args, "-vvv")
```

Comment:
There's no such thing as `--verbose-max` for now (maybe that's something we can add in the future). The way it works, each `v` gives you more logs, and over 3 `v`s, you get the `Trace` type of logs.
This is not expected to change, but we didn't reach 1.0.0 yet, so for the time being, we'll have to adapt Kata is anything changes in Cloud-Hypervisor (which of course we're trying to avoid as much as possible :))